torax/plotting/configs/global_params_plot_config.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2024 DeepMind Technologies Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Plotting configuration for global plasma parameters vs time."""
+
+from torax.plotting import plotruns_lib
+
+PLOT_CONFIG = plotruns_lib.FigureProperties(
+    rows=2,
+    cols=3,
+    tick_fontsize=8,
+    axes_fontsize=8,
+    default_legend_fontsize=7,
+    figure_size_factor=5,
+    title_fontsize=12,
+    axes=(
+        # For chi, set histogram percentile for y-axis upper limit, due to
```

Comment:
This comment can be removed

Reply:
Done [4929853](https://github.com/google-deepmind/torax/pull/617/commits/4929853bac3e43317d88aea63c0fd326787d51dd).